plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -97,7 +97,7 @@ class ServicesTable extends React.Component {
     if (
       actionItem.id !== EDIT &&
       actionItem.id !== DELETE &&
-      (containsSDKService || isSDKService(service)) &&
+      containsSDKService &&
```

Comment:
Assumption: this check is moved into `Framework.js`, because SDK packages _always_ are `Framework`.